packages/terra-form-checkbox/src/Checkbox.scss

```diff
@@ -0,0 +1,116 @@
+:local {
+  .control {
+    color: var(--terra-form-checkbox-font-color, rgb(0, 0, 0));
+    display: inline-block;
+    font-family: inherit;
+    margin-right: var(--terra-form-checkbox-margin-right, 2em);
+    margin-top: var(--terra-form-checkbox-margin-top, 0.625em);
+    vertical-align: middle;
+
+    .label-text {
+      font-size: var(--terra-form-checkbox-font-size, 1rem);
+    }
+
+    /* Hide actual input element */
+    input:not(:checked),
+    input:checked {
```

Comment:
You can just use `display: none` to hide the input.

Reply:
The custom css checkbox depends on interaction with the actual input element, so I opted to go with the hidden positioning route found in html5-boilerplat: https://github.com/h5bp/html5-boilerplate/blob/c3d52666d0d297e50f965af4ad214702a683e02f/src/css/main.css#L126-L135

Reply:
If your goal is to allow the consumer to interact with the actual input, this approach doesn't work either. With a 1px height and width the click point is too small, and it is placed behind the label so all click events will hit the label, not the input. The reason your controls work now is that all your examples have an id which allows interacting with the label to affect the state of the input. If you had an example without an id it wouldn't work. (Note: This also means that id probably needs to be a required prop).

Reply:
Also note, for visually hidden elements you can just use `visibility: hidden`

Reply:
Anyone have thoughts on making `id` required? Another route is changing the implementation back to the old Control, with `input` inside `label`. `label` will automatically handle `input` toggling.
``` html
<label>
  <input></input>
  <span></span>
</label>
```